src/main/java/com/dev/cinema/model/dto/shoppingcart/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,35 @@
+package com.dev.cinema.model.dto.shoppingcart;
+
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.model.User;
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Ticket> tickets;
+    private User user;
```

Comment:
```suggestion
    private List<Long> ticketIds;
```
I believe cartId and userId are the same?